db/migrate/20161226201452_change_tasks_timestamps_default_to_null.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeTasksTimestampsDefaultToNull < ActiveRecord::Migration
+  def change
+    change_column_null :tasks, :updated_at, false
+    change_column_null :tasks, :created_at, false
+  end
+end
```

Comment:
I had already created the Task model and then realized it would probably be best to be able to have timestamps for them. As for keeping the nullability false, I had read that this was best practice so you didn't end up with null timestamp tables in your database.